source/include/azure_iot_hub_client.h

```diff
@@ -36,11 +36,15 @@ typedef enum AzureIoTHubMessageQoS
 
 typedef enum AzureIoTHubMessageType
 {
-    eAzureIoTHubCloudToDeviceMessage = 1,    /**< The message is a cloud message. */
-    eAzureIoTHubDirectMethodMessage,         /**< The message is a direct method message. */
-    eAzureIoTHubTwinGetMessage,              /**< The message is a twin get response (payload contains the twin document). */
-    eAzureIoTHubTwinReportedResponseMessage, /**< The message is a twin reported property status response. */
-    eAzureIoTHubTwinDesiredPropertyMessage,  /**< The message is a twin desired property message (incoming from the service). */
+    eAzureIoTHubCloudToDeviceMessage = 1,          /**< The message is a cloud message. */
+    eAzureIoTHubDirectMethodMessage,               /**< The message is a direct method message. */
+    eAzureIoTHubCommandMessage,                    /**< The message is a command message. */
+    eAzureIoTHubTwinGetMessage,                    /**< The message is a twin get response (payload contains the twin document). */
+    eAzureIoTHubTwinReportedResponseMessage,       /**< The message is a twin reported property status response. */
+    eAzureIoTHubTwinDesiredPropertyMessage,        /**< The message is a twin desired property message (incoming from the service). */
+    eAzureIoTHubPropertiesGetMessage,              /**< The message is a property get response (payload contains the property document). */
+    eAzureIoTHubPropertiesReportedResponseMessage, /**< The message is a reported property status response. */
+    eAzureIoTHubPropertiesDesiredPropertyMessage,  /**< The message is a desired property message (incoming from the service). */
```

Comment:
these and the command are the added types

Reply:
can we just not remove the older names? I am just wondering what happens people used method in application code and we always stamp message in callback as command?